q2_gneiss/plot/tests/test_plot.py

```diff
@@ -336,6 +342,41 @@ def test_balance_taxonomy_multi_categorical(self):
             self.assertIn('Denominator taxa', html)
             self.assertNotIn('Proportion', html)
 
+    def test_balance_taxonomy_partial_numerical_categorical(self):
```

Comment:
Maybe you can refactor this to loop over a list of all the files to check the presence of?

Reply:
For this test, I basically just copy-pasted the other tests in the file that also used valid metadata columns. So this test looks identical to them, but it uses the new metadata. Should I still refactor specifically this test? Should I refactor all tests that behave in this manner? Or should I leave it as is?

Reply:
> Should I still refactor specifically this test?

Yes, please.

>  Should I refactor all tests that behave in this manner?

No need, thanks though.